GridMaker_Sketch.py

```diff
@@ -11,12 +13,22 @@ def __init__(self, **entries):
 
 
 def readJsonFile(file):
+    if not file:
+        raise ValueError("File not found or file pattern did not match.")
     with open(file, 'r') as openFile:
         return json.load(openFile)
 
 
-qualiFile = "Qualifying.json"
-entryListFile = "EntryList.json"
+qualiFile = None
+for file in os.listdir('.'):
+    if fnmatch.fnmatch(file, "*_Q.json"):
```

Comment:
Going with the first match of any json file ending in _Q will save the user having to rename the file, but also potentially leads to issues if the user hasn't cleaned up after previous sessions and have left the last _Q file in place. Not sure if this will actually lead to a better UX or if it will cause more headaches for users potentially...